code/game/objects/items/weapons/twohanded.dm

```diff
@@ -511,4 +512,45 @@
 	return 0
 
 /obj/item/weapon/twohanded/vibro_weapon/update_icon()
-	icon_state = "hfrequency[wielded]"
\ No newline at end of file
+	icon_state = "hfrequency[wielded]"
+
+/*
+ * Bone Axe
+ */
+/obj/item/weapon/twohanded/fireaxe/boneaxe  // Blatant imitation of the fireaxe, but made out of bone.
+	icon_state = "bone_axe0"
+	name = "bone axe"
+	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+	force_wielded = 23
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/update_icon()
+	icon_state = "bone_axe[wielded]"
+
+/*
+ * Bone Spear
+ */
+/obj/item/weapon/twohanded/bonespear	//Blatant imitation of spear, but made out of bone. Not valid for explosive modification.
+	icon_state = "bone_spear0"
+	name = "bone spear"
+	desc = "A haphazardly-constructed yet still deadly weapon. The pinnacle of modern technology."
+	force = 11
+	w_class = 4
+	slot_flags = SLOT_BACK
+	force_unwielded = 11
+	force_wielded = 21					//I have no idea how to balance
+	throwforce = 22
+	throw_speed = 4
+	embedded_impact_pain_multiplier = 3
+	armour_penetration = 15				//Enhanced armor piercing
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "poked", "jabbed", "torn", "gored")
+	sharpness = IS_SHARP
+
+/obj/item/weapon/twohanded/bonespear/update_icon()
+		icon_state = "bone_spear[wielded]"
+
+/obj/item/weapon/twohanded/bonespear/afterattack(atom/movable/AM, mob/user, proximity)
```

Comment:
It's literally the same as the spear. Apparently necessary to make it hit nearby only? If you're complaining about this then you have to complain about the vanilla spear, okay, because this is a replica.